My Simple App/Application/UnknownTableViewCell.swift

```diff
@@ -9,18 +9,49 @@
 import UIKit
 
 class UnknownTableViewCell: UITableViewCell {
+  var titleLabel: UILabel = {
+    let label = UILabel()
+    // When you're writing programatic UI using anchors, this property has to
+    //   be disabled on all views. At larger dev shops, there will likely be some sort
+    //   of view framework and system views (eg UILabel, UIButton) will not be used
+    //   directly. The wrapper views will disable this property for you in that case.
+    label.translatesAutoresizingMaskIntoConstraints = false
+    label.font = .systemFont(ofSize: 17)
+    return label
+  }()
+  
+  override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+    super.init(style: style, reuseIdentifier: reuseIdentifier)
+    setupViews()
+    setupLayout()
+  }
+  
+  required init?(coder: NSCoder) {
```

Comment:
```suggestion
  @available(*, unavailable)
  required init?(coder: NSCoder) {
```
nit: add unavailable to throw a compile-time error (even though this does not get called until runtime). This will also make the method not required if someone subclasses this ViewController